gateway-service-baseline-lib/build.gradle.kts

```diff
@@ -15,8 +15,11 @@ dependencies {
   // Common utilities
   implementation("org.apache.commons:commons-lang3:3.10")
   implementation("org.apache.commons:commons-math:2.2")
-  implementation("com.google.protobuf:protobuf-java-util:3.17.3")
+  implementation("com.google.protobuf:protobuf-java-util:3.19.2")
   constraints {
+    implementation("com.google.protobuf:protobuf-java:3.19.2") {
```

Comment:
We want to avoid constraints when possible and instead fix the source issues. Went through all the deps in https://github.com/hypertrace/gateway-service/pull/118 to separate those issues out of this PR.